src/pages/Area51Page.tsx

```diff
@@ -0,0 +1,52 @@
+import "./Area51Page.scss";
+import imageSecret from "../assets/image-martian-selfie.jpeg";
+import { useState, useEffect } from "react";
+
+const checkSecurityCookie = () => {
+  const cookieName = "security-clearance";
+  const cookies = document.cookie.split(";");
+  for (let i = 0; i < cookies.length; i++) {
+    const cookie = cookies[i].trim();
+    if (cookie.startsWith(`${encodeURIComponent(cookieName)}=`)) {
+      const [, cookieValue] = cookie.split("=");
+      const cookieVal = decodeURIComponent(cookieValue);
+      return cookieVal === "super-secret-cleared";
+    }
+  }
+  return false;
+};
+
+function Area51Page() {
+  const [securityCleared, setSecurityCleared] = useState<boolean>();
+
+  useEffect(() => {
+    const cookieVal = checkSecurityCookie();
+    setSecurityCleared(cookieVal);
+  }, []);
+
+  return securityCleared ? (
+    <div className="Area51Page">
+      <h1>Area 51.</h1>
+      <h2 className="Area51Page__classification">TOP SECRET</h2>
```

Comment:
Semantically, this isn't a header
```suggestion
      <p className="Area51Page__classification">TOP SECRET</p>
```